src/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './css/index.css';
+// import './css/main.css';
+// import './css/animate.css';
```

Comment:
Never leave a commented code going to production. You can always return back to it from the history.